### ISSUE TITLE
How to prevent line break caused by slash (/) when parsing CSV file?

### ISSUE THREAD

**Issue by luizfp** [uniVocity/univocity-parsers#272]
Hi, guys. 

I have been using this library for a while now and I'm currently facing a weird problem.
If my CSV file has, for instance, the following row:

333 | mmm1234 | 21/08/2018 | 5.000,50 | Test | Test | 28.98 | 18.98
-- | -- | -- | -- | -- | -- | -- | --

Every slash (**/**) in the Date (21/08/2018) causes a line break when we call `parser.parseAll(file)`.
We ended up with the following:
![image](https://user-images.githubusercontent.com/7897981/44463896-93c92580-a5ef-11e8-9000-8bf93f384248.png)

I have searched the [docs](https://www.univocity.com/pages/univocity_parsers_csv.html#working-with-csv) but have not find how to prevent this line break from happen.

How could we prevent this?

EDIT:
--------
Library version:
```XML
<dependency>
	<groupId>com.univocity</groupId>
	<artifactId>univocity-parsers</artifactId>
	<version>2.6.3</version>
	<type>jar</type>
</dependency>
```

Parse settings:
```java
final CsvParserSettings settings = new CsvParserSettings();
settings.setDelimiterDetectionEnabled(true);
settings.setHeaderExtractionEnabled(true);
final CsvParser parser = new CsvParser(settings);
final List<String[]> rows = parser.parseAll(file);
```

**Comment by jbax** [comment]
That's a very weird one. What settings are you using? Can you put this in a
unit test I could run to reproduce the error?

**Comment by luizfp** [comment]
Sure.

I created a simple repository with only the test class and the csv file: [https://github.com/luizfp/univocitylinebreakproblem](https://github.com/luizfp/univocitylinebreakproblem)

The test is simple:
```java
import com.univocity.parsers.csv.CsvParser;
import com.univocity.parsers.csv.CsvParserSettings;
import org.junit.Test;

import java.io.File;
import java.util.List;

import static org.junit.Assert.assertEquals;

/**
 * Created on 22/08/2018
 *
 * @author Luiz Felipe (https://github.com/luizfp)
 */
public class LineBreakProblemTest {
    private static final String TARGET_PATH = System.getProperty("user.dir");
    private static final String FILE_NAME = "file_test.csv";

    @Test
    public void testLineBreak() {
        final File file = new File(TARGET_PATH, FILE_NAME);
        final CsvParserSettings settings = new CsvParserSettings();
        settings.setDelimiterDetectionEnabled(true);
        settings.setHeaderExtractionEnabled(true);
        final CsvParser parser = new CsvParser(settings);
        final List<String[]> rows = parser.parseAll(file);
        assertEquals(1, rows.size());
        assertEquals(1, rows.get(0).length);
    }
}
```

Glad to help!

**Comment by jbax** [comment]
That was a great help, thank you.

I've fixed and released a 2.7.6-SNAPSHOT build that you can use to test. You can follow the [instructions here](https://www.univocity.com/pages/univocity_parsers_download#snapshot-releases) to get it

Basically the parser was detecting the delimiter as '/'. The detection process is not very reliable when a small input with one or two rows is provided.

What you can also do on the current version you are using is to give the parser a list of allowed delimiters:

`settings.setDelimiterDetectionEnabled(true, ',', ';',... and any other character that can be a valid delimiter for the inputs you are handling);`

Muito obrigado por usar o nosso parser. Eh sempre um prazer falar com um conterraneo. Abracos
